scripts/cache_artifacts.sh

```diff
@@ -14,7 +14,11 @@
 
 set -e
 
-readarray -d '' metas < <(find "$1" -name 'meta.yaml' -print0)
+if [[ $2 == "--latest-only" ]]; then
```

Comment:
This shouldn't be necessary if we clean non-latest metas from the filesystem in the builder image.

Reply:
True, but this allows another option. Maybe you want to keep all the meta.yaml files but only CACHE the OFFLINE versions for the latest ones.

Reply:
Fair enough.